clouddriver-security/src/main/groovy/com/netflix/spinnaker/clouddriver/security/resources/Global.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.security.resources;
+
+/**
+ * Marker interface indicating that a description is global and may not have account-level credentials specified.
+ */
+public interface Global {
```

Comment:
`Global` isn't really descriptive of what it is - a credential-less operation? I'm not familiar enough with what you're working on to know what kind of auth impact it will have.